### ISSUE TITLE
Search of filename in Directory

### ISSUE THREAD

**Issue by 3797kaushik** [notepad-plus-plus/notepad-plus-plus#4939]
### Description of the Issue
When we open a project as workspace and want to search some file there is no functionality like searach of file or suggesting the realated file names. Though there are third party libraries for this but they are not user friendly.

### Expected Behavior
When we open folder with multiple folder and files as workspace There should be a search of file functionality and it should also suggest the names similar to search functionality inside text,

**Comment by mirh** [comment]
I'm relatively sure you can use regex in the "Filters" field. See #2433

**Comment by 3797kaushik** [comment]
> I'm relatively sure you can use regex in the "Filters" field. See #2433

Yeah I have checked the filters but it is to search within the file. But my Request is to search the file suppose if we are working on 1000's of files through the files are sorted we need to scroll scrollbar and locate the file which takes lot of time.

**Comment by kavika13** [comment]
Most other modern editors have this either natively out of the box, as their default "open file" experience (or at least one of such UIs), or they at least have it as a plug-in.

TBH pretty surprised this is missing.
